imagestreams/valkey-centos.json

```diff
@@ -0,0 +1,68 @@
+{
+  "kind": "ImageStream",
+  "apiVersion": "image.openshift.io/v1",
+  "metadata": {
+    "name": "valkey",
+    "annotations": {
+      "openshift.io/display-name": "Valkey"
+    }
+  },
+  "spec": {
+    "tags": [
+      {
+        "name": "6-el9",
+        "annotations": {
+          "openshift.io/display-name": "Valkey 6 (CentOS Stream 9)",
+          "openshift.io/provider-display-name": "Red Hat, Inc.",
+          "description": "Provides a Valkey 6 database on CentOS Stream 9. For more information about using this database image, including OpenShift considerations, see  https://github.com/sclorg/valkey-container/tree/master/6/README.md.",
```

Comment:
There are some versions that will never exist, the templates will need some more changes. Or remove entirely if the helm charts are the only way forward (something to consider).